server/src/services/infrastructure/scheduler.ts

```diff
@@ -14,3 +17,22 @@ export const checkCovid19Updates = () => {
         );
     });
 };
+
+export const sendReleaseNotificationToUsers = async (bot: TelegramBot) => {
+    // At 08:00 PM, only on Monday
+    schedule.scheduleJob('0 0 20 * * 1', async () => {
+        const [err, users] = await catchAsyncError(getTelegramAllUsers());
+        if (err) {
+            logger.error(
+                'sendReleaseNotificationToUsers failed when accessing users Db',
+                err,
+                LogCategory.Scheduler
+            );
+            return;
+        }
+
+        for (const usr of users) {
+            bot.sendMessage(usr.chatId, 'Hello DUDE, What\'s UP? ');
```

Comment:
should we merge it? :D

Reply:
Your question is super fair.
I've just made message more configurable